test/tree-sitter-corpus.jl.faceup

```diff
@@ -0,0 +1,784 @@
+«x:# ==============================»
+«x:# tuple collections»
+«x:# ==============================»
+
+()
+«x:# There's no (,)»
+
+(«c:1») «x:# NOT a tuple»
+(«c:1»,)
+(«c:2»,«c:3»,«c:4»,)
+
+«x:# ==============================»
+«x:# named tuple collections»
+«x:# ==============================»
+
+(«v:a» = «c:1») «x:# NOT a tuple»
+(a = «c:1»,)
+(a = «c:1», b = «c:2»)
+(;)
+(; a)
+(; a = «c:1»)
+(; a = «c:1», b = «c:2»)
+(; a, foo.b)
+
+«x:# ==============================»
+«x:# vector array collections»
+«x:# ==============================»
+
+[]
+«x:# There's no [,]»
+[x]
+[x,]
+[«c:1», «c:2»]
+
+«x:# Check unary-and-binary-operators»
+[x.-y, «c:2»]
+
+«x:# ==============================»
+«x:# matrix array collections»
+«x:# ==============================»
+
+[x;]
+[«c:1» «c:2»]
+[«c:1» «c:2»; «c:3» «c:4»]
+[«c:1» «c:2»
+ «c:3» «c:4»]
+[«c:1»
+ «c:2»
+ «c:3»
+]
+[
+ a;
+ b;
+ c;
+]
+[«c:1»;; «c:2»;; «c:3»;;; «c:4»;; «c:5»;; «c:6»;;]
+Int[«c:1» «c:2» «c:3» «c:4»]
+
+«x:# ========================================»
+«x:# comprehension array collections»
+«x:# ========================================»
+
+[x «k:for» x «k:in» xs]
+[x
+  «k:for» x «k:in» xs
+  «k:if» x > «c:0»
+]
+UInt[b(c, e) «k:for» c «k:in» d «k:for» e «k:in» f]
+
+f(«c:1», «c:2», i «k:for» i «k:in» iter)
+(b(c, e) «k:for» c «k:in» d, e = «c:5» «k:if» e)
+
+«x:# ==============================»
+«x:# module definitions»
+«x:# ==============================»
+
+«k:module» A
+
+«k:baremodule» B «k:end»
+
+«k:module» C
+«k:end»
+
+«k:end»
+
+«x:# ==============================»
+«x:# abstract type definitions»
+«x:# ==============================»
+
+«k:abstract» «k:type» «t:T» «k:end»
+«k:abstract» «k:type» «t:T» <: «t:S» «k:end»
+«k:abstract» «k:type» «t:T»{«t:S»} <: «t:U» «k:end»
+
+«x:# ==============================»
+«x:# primitive type definitions»
+«x:# ==============================»
+
+«k:primitive» «k:type» «t:T» «c:8» «k:end»
+«k:primitive» «k:type» «t:T» <: «t:S» «c:16» «k:end»
+«k:primitive» «k:type» «t:Ptr»{«t:T»} «c:32» «k:end»
+
+«x:# ==============================»
+«x:# struct definitions»
+«x:# ==============================»
+
+«k:struct» «t:Unit» «k:end»
+
+«k:struct» «t:MyInt» field::«t:Int» «k:end»
+
+«k:mutable» «k:struct» «t:Foo»
```

Comment:
I don't think there's any world in which it makes a difference, but arguably the space between "mutable" and "struct" is part of the single keyword "mutable struct".

Reply:
The Julia tree-sitter grammar itself reports "mutable" and "struct" as two separate keywords. I'm not sure if a tree-sitter query can be written such that it fontifies these two keywords together. Even if it could, it would make the query more complicated, arguably unnecessarily so. Since it wouldn't provide any visual benefit, I'd rather not mess with it. WDYT?

Reply:
Sounds reasonable to me. No need to complicate things for exactly no benefit.